### PR TITLE
HipChat -> Slack, create Slack channel on prio1s and invite relevant people

### PR DESCRIPTION
**Pull request by abesto**
Each commit is a self-contained change, reviewing may be easiest by going commit to commit. They kind of depend on each other, so I'll just open a PR for the work I have so far instead of opening six tiny PRs.

### CODE REVIEW

**Comment by abesto** [comment]
@kbence I can have eyes on this? :)

File: pasha_modules/util.coffee [hunk 1]
```diff
@@ -6,32 +6,23 @@ constant = require('../pasha_modules/constant').constant
...
-        scribeLog "ERROR downloadUsers #{error}"
-        setUsersCallback([])
+    slackApi("users.list", {token: constant.slackApiToken}, (err, res, data) ->
+        if err || !data.ok
+          scribeLog "ERROR downloadUsers #{err || data.error}"
+          setUsersCallback([])
+        else
+          setUsersCallback(data.members)
+          scribeLog "downloaded #{data.members.length} users"
+    )
```
Comment (kbence) [inline pasha_modules/util.coffee:22]: Slight difference in our style (feel free to ignore this comment): I'd rather remove the outer parentheses.

Comment (kbence) [inline pasha_modules/util.coffee:22]: And a nice piece of code, needless to say.

File: scripts/pasha_main.coffee [hunk 2]
```diff
@@ -246,9 +267,22 @@ module.exports = (robot) ->
...
+                if !err && data.ok
+                    pashaState.prio1.channel = {id: data.channel.id, name: channelName}
+                    robot.brain.set(constant.pashaStateKey, JSON.stringify(pashaState))
+                    msg.send("Created channel ##{channelName}, please join and keep all prio1 communication there.")
+                    invitePrio1RolesToPrio1SlackChannel()
+                else
+                    msg.send("Failed to create channel #{channelName}: #{err || data.error}")
+
+            )
+
```
Comment (kbence) [inline scripts/pasha_main.coffee:283]: Do I understand correctly? Our intention is to keep the prio1 channel until we need it and archive it manually later? Great feature by the way.

**Comment by kbence** [comment]
Big like for the feature and the fixes. I want this in `master`!
:shipit:

File: scripts/pasha_main.coffee (hunk 2 above)
Comment (abesto) [inline scripts/pasha_main.coffee:283]: That's correct. The only place where Pasha could meaningfully close the channel would be `prio1 stop`, but we usually still have more conversations / data to share after the outage is over, so I'd say let's do it manually.

**Comment by abesto** [comment]
Thanks for the prompt review!

File: scripts/pasha_main.coffee (hunk 2 above)
Comment (kbence) [inline scripts/pasha_main.coffee:283]: Possible improvement for the future: archive it automatically after x hours of silence :).

Comment (abesto) [inline scripts/pasha_main.coffee:283]: That's a good one, open an issue to track it?

**Pull request merged**